src/core/PluginInstaller.ts

```diff
@@ -33,7 +34,7 @@ while (topModule.parent) {
 export default class PluginInstaller {
   private readonly pluginDir: string;
   readonly require: (name: string) => any = topModule.require.bind(topModule);
-  private readonly resolve = (request: string) => (module.constructor as any)._resolveFilename(request, topModule);
+  private readonly resolve = (request: string) => resolve.sync(request);
```

Comment:
Question about this, the current form `(module.constructor as any)._resolveFilename(request, topModule);` ensures that the resolution is done from the point of view of the topmost module (the main app). This is necessary because if for example you have skwalking `npm link`ed into a project instead of installed alongside all its dependancies then normal resolve will get the wrong path, one that sits in the skywalking node_modules path instead of the applicatiuon. Does the `resolve` package do this?

Reply:
Sorry, I missed this point. According to the documentation [resolve](https://www.npmjs.com/package/resolve), this can be done by 
```private readonly resolve = (request: string) => resolve.sync(request, {basedir: path.dirname(topModule.id) });```
 if You agree i will update later

Reply:
Ok, well I checked your fix and it seems to work, so I am ok with this.